backend/app/domains/users/routers.py

```
@@ -0,0 +1,136 @@
+
+        if user_crud.get_user_by_email(user_data.email):
+            raise HTTPException(status_code=400, detail="Email already exists")
+
+        # Create user
+        user = user_crud.create_user(user_data)
+        if not user:
+            raise HTTPException(status_code=500, detail="Failed to create user")
+
+        return {"message": "User registered successfully", "user_id": user.id}
+    except Exception as e:
+        logger.error(f"register_student에서 예외 발생: {e}", exc_info=True)
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)